contrib/previewers/lib/basisu_transcoder.h

```diff
@@ -80,7 +80,7 @@
 		#if defined(_DEBUG) || defined(DEBUG)
 			// This is madness, but we need to disable iterator debugging in debug builds or the encoder is unsable because MSVC's iterator debugging implementation is totally broken.
 			#ifndef _ITERATOR_DEBUG_LEVEL
-			#define _ITERATOR_DEBUG_LEVEL 1
+			#define _ITERATOR_DEBUG_LEVEL 2
```

Comment:
Will this header be consumed directly? If not, it may be better to just define BASISU_NO_ITERATOR_DEBUG_LEVEL in the project lua file instead of changing source. Changing source makes it more tedious to upgrade.